alvr/filesystem-layout/src/lib.rs

```diff
@@ -43,21 +46,29 @@ lazy_static! {
             presets_dir: PathBuf::from("presets"),
             vrcompositor_wrapper: PathBuf::from(""),
             launcher_exe: PathBuf::from("ALVR Launcher.exe"),
-            dashboard_resources_dir: PathBuf::from("share/alvr/dashboard"),
+            dashboard_resources_dir: PathBuf::from("dashboard"),
         }
-    } else {
+    } else if cfg!(target_os = "linux") {
         Layout {
             openvr_driver_dir: PathBuf::from("lib64/alvr"),
             presets_dir: PathBuf::from("share/alvr/presets"),
             vrcompositor_wrapper: PathBuf::from("libexec/alvr/vrcompositor-wrapper"),
             launcher_exe: PathBuf::from("bin/alvr_launcher"),
             dashboard_resources_dir: PathBuf::from("share/alvr/dashboard"),
         }
+    } else {
```

Comment:
Maybe make this macOS and then the last one panic/unimplemented? There are more platforms.

Reply:
I mean, I don't know anyone planning to build this on BSD or Solaris 🙂